terminology/common-terminology.md

```diff
@@ -74,20 +74,32 @@ Various CCC projects will deliver code aimed at one or more of the packaging mod
 
 * Gramine could be utilized in the role of the “TEE Shim” across any package on SGX.
  
-## Package Deployability Prerequisites
+## Attestation
+
+However a workload is packaged, the resulting deployment should support interaction in a way that
+proves it is running in a TEE.
+
+While in traditional computing, there’s often a desire to authenticate a server, in Confidential
+Computing the goal is to authenticate the TEE, and attestation is the means to do that.
+In Confidential Computing an attestation is a hardware signed report of the measurements of the TCB.
+The measurements provided in the attestation relate to the TCB boundaries
+depicted in the diagram above. An attestation for process isolation then authenticates an
+application whereas an attestation for VM isolation conceptually authenticates a VM or the virtual
+firmware used to launch the VM.
```

Comment:
Above is using "attestation" as a synonym for "evidence".  I'll observe that IETF RATS never uses it that way, so not sure we should either.

Reply:
Discussion: is there a sentence in the RATS architecture that could be borrowed or adapted?  Maybe the CCC-Attestation SIG could come up with a sentence?